notify-debouncer-full/src/lib.rs

```diff
@@ -271,6 +271,10 @@ impl<T: FileIdCache> DebounceDataInner<T> {
             return;
         }
 
+        if event.paths.is_empty() {
+            log::warn!("add_event: Skipping event with empty paths array");
```

Comment:
```suggestion
            log::warn!("skipping event with no paths: {event:?}");
```

I would rephrase this message a bit. But more importantly, print the event. The event may have already been logged a few lines above, but only with `trace` level.